### PR TITLE
feat: Implement multilingual for site properties - EXO-65344 - Meeds-io/MIPs#72

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteManagementRestService.java [hunk 1]
```diff
@@ -183,10 +190,13 @@ public Response updateSitePermissions(@Context
...
       PortalConfig portalConfig = layoutService.getPortalConfig(new SiteKey(siteType, siteName));
-      return Response.ok(EntityBuilder.buildSiteEntity(portalConfig, request, false, false)).build();
+      return Response.ok(EntityBuilder.buildSiteEntity(portalConfig, request, false, false, getLocale(lang))).build();
     } catch (Exception e) {
       LOG.error("Error when updating site permissions with name {} and type {}", siteName, siteType, e);
       return Response.serverError().build();
     }
   }
+    private Locale getLocale(String lang) {
+        return org.apache.commons.lang3.StringUtils.isBlank(lang) ? null : Locale.forLanguageTag(lang);
```
Comment (azayati) [inline layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteManagementRestService.java]: use org.apache.commons.lang.StringUtils instead

File: layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteManagementPortlet_en.properties [hunk 2]
```diff
@@ -25,4 +25,6 @@ siteManagement.label.updateSite.unauthorized=You do not have the rights to edit
 siteManagement.label.displayOrder=Display order
 siteManagement.label.displayed=The site is available in the left navigation of the site dw in position :
 siteManagement.label.notDisplayed=The site isn't available in left navigation of the site dw
-siteManagement.displayOrder.error=Invalid value
\ No newline at end of file
+siteManagement.displayOrder.error=Invalid value
+siteManagement.form.translateTitle=Add translation for Site label
+siteManagement.form.translateDescription=Add translation for Site description
```
Comment (azayati) [inline layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteManagementPortlet_en.properties]:
```suggestion
siteManagement.label.translateTitle=Add translation for Site label
siteManagement.label.translateDescription=Add translation for Site description
```

File: layout-management-webapps/src/main/webapp/vue-app/site-management/components/SiteCardPropertiesDrawer.vue [hunk 3]
```diff
@@ -124,13 +150,19 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 export default {
   data() {
     return {
+      drawer: false,
```
Comment (azayati) [inline layout-management-webapps/src/main/webapp/vue-app/site-management/components/SiteCardPropertiesDrawer.vue]:
```suggestion
      siteCardPropertiesDrawer: false,
```

File: layout-management-webapps/src/main/webapp/vue-app/site-management/components/SiteCardPropertiesDrawer.vue [hunk 4]
```diff
@@ -139,35 +171,44 @@ export default {
...
+  },
   computed: {
     saveDisabled(){
       return !this.siteLabel || (this.displayed && this.displayOrder < 1);
     },
     displayedDisabled(){
       return this.site?.metaSite;
     },
+    showDrawerContent() {
+      return this.drawer  && !!this.site;
```
Comment (azayati) [inline layout-management-webapps/src/main/webapp/vue-app/site-management/components/SiteCardPropertiesDrawer.vue]:
```suggestion
      return this.drawer && !!this.site;
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**